queries/php_only/highlights.scm

```diff
@@ -265,12 +286,17 @@
   "protected"
   "public"
   "readonly"
-  "static"
+  (static_modifier)
+  (function_static_declaration
+    "static")
```

Comment:
Hmm, why

Reply:
Shouldn't this be capturing static

Reply:
Because static is used for many things---type.builtin, variable.builtin, and keyword.modifer.